All Core Devs Meetings/meeting 94.md

```diff
@@ -0,0 +1,437 @@
+# All Core Devs Meeting 94 Notes
+### Meeting Date/Time: Friday, Aug 21 2020, 14:00 UTC
+### Meeting Duration: 1:32 hrs
+### [GitHub Agenda](https://github.com/ethereum/pm/issues/200)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=q6bIeSu7r9o&t=2835s)
+### Moderator: Hudson Jameson
+### Notes: Alita Moore
+  
+---
+  
+# Summary
+  
+## EIP Status
+EIP | Status
+  
+  
+## Decisions Made
+  
+Decision Item | Description
+--|--
+**94.1.1**: Discuss the possibility of launching a Prog POW on a test net next week in line with the Ben's compromise
+**94.1.2**: Move further discussion on precompile to ETH Magicians forum [here](https://ethereum-magicians.org/t/evm384-feedback-and-discussion/4533)
+  
+  
+  
+## Actions Required
+  
+Action Item | Description
+--|--
+**94.2.1**: Decide, by next meeting, whether to go with Axic's and update the sources that say we're going with EIP 2537
+**93.2.2**: Vitalik to start coordinating a writeup on the impact of the gas limit increase (what will it break)
+**93.2.3**: revisit the discussion on EIP 384
+  
+  
+## Helpful Links
+Link | Description
+--|--
+**94.3.1**: [EIP 2666 analysis results](https://github.com/ethereum/pm/issues/200#issuecomment-678115862)
+**93.3.2**:  [Context on 1884](https://github.com/holiman/eip-1884-security/)
+  
+---
+  
+# Agenda
+  
+<!-- MDTOC maxdepth:6 firsth1:1 numbering:0 bullets:1 updateOnSave:1 -->
+- [1. Progress on EIP-1057](#-1)
+- [2. Ewasm Team: Findings on "evm384"](#-2)
+- 3 EIP Discussion
+    - [3a. Vitalik's Idea](#3)
+- [4. YOLO / YOLOv2 & Berlin state tests update](#4)
+- [5. Raw benchmarks for EIP 2666](#5)
+- [6. EIPIP Meeting: New EIP Statuses](#6)
+- [7. Ethereum Cat Herders Survey Results](#7)
+- [8. Migration from ACD Gitter to Eth R&D Discord](#8)
+- [9. Review previous decisions made and action items (if notes available)](#9)
+  
+---
+  
+# 1 Progress on EIP-1057
+  
+Video | [3:56](https://youtu.be/q6bIeSu7r9o?t=236)
+  
+**Hudson** - Hello and welcome to a theorem poor developer meeting No. 94, I'm your host, Hudson, and we're going to go to the first item on the agenda. Greg wants to give an update on the progress of EIP 10 57. So, Greg, if you want to admit and go through that real quick, that'd be great.
```

Comment:
Hello and welcome to a Ethereum Core Developer meeting No. 94, I'm your host, Hudson, and we're going to go to the first item on the agenda. Greg wants to give an update on the progress of EIP 10 57. So, Greg, if you want to admit and go through that real quick, that'd be great.